spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/cassandra/CassandraHealthIndicator.java

```diff
@@ -38,25 +39,24 @@
 	private static final SimpleStatement SELECT = SimpleStatement
 			.newInstance("SELECT release_version FROM system.local").setConsistencyLevel(ConsistencyLevel.LOCAL_ONE);
 
-	private CassandraOperations cassandraOperations;
-
-	public CassandraHealthIndicator() {
```

Comment:
Removed because it doesn't buy us anything, but this is a breaking change. Let me know if you prefer to keep it.